src/cassandra.lua

```diff
@@ -74,6 +74,12 @@ local function startup(self)
   return true
 end
 
+local function split(str)
```

Comment:
Yeah I wanted to rename it and wasn't inspired, I guess split_by_port makes sense.